src/steadystateproblem.cpp

```diff
@@ -24,7 +24,7 @@ SteadystateProblem::SteadystateProblem(const Solver &solver, const Model &model)
       xdot(model.nx_solver), xdot_old(model.nx_solver),
       sx(model.nx_solver, model.nplist()), sdx(model.nx_solver, model.nplist()),
       dJydx(model.nJ * model.nx_solver * model.nt(), 0.0), numsteps(3, 0),
-      numlinsteps(solver.getNewtonMaxSteps() * 2, 0) {}
+      numlinsteps(std::max(solver.getNewtonMaxSteps(), 1) * 2, 0) {}
```

Comment:
Why not have maxsteps as member of SteadystateProblem and enforcing >=1 there instead of using max in 5 places?

Reply:
Yes, looks nicer...
Thanks!